tests/neg/i13044.check

```diff
@@ -1,3 +1,15 @@
+-- [E195] Potential Issue Warning: tests/neg/i13044.scala:26:8 ---------------------------------------------------------
+26 |        new Schema[A] {
+   |        ^
+   |        New anonymous class definition will be duplicated at each inline site
+   |
+   | longer explanation available when compiling with `-explain`
+-- [E195] Potential Issue Warning: tests/neg/i13044.scala:32:8 ---------------------------------------------------------
+32 |        new Schema[A] {
+   |        ^
+   |        New anonymous class definition will be duplicated at each inline site
+   |
+   | longer explanation available when compiling with `-explain`
```

Comment:
It is better to replace the anonymous class in the [test](https://github.com/scala/scala3/blob/main/tests/neg/i13044.scala) than adding the warnings here. You can do that by simply replacing them by `???`

Reply:
For this particular test, I thought it was necessary to keep it as is since replacing those line with ??? will cause the compilation to pass. I could add @nowarn as an alternative to modifying the checkfile, but I don't know the exact context in which the test was constructed so I am hesitant to change it too significantly in case it ends up passing but not testing what it was intended to test.

Reply:
Mmmh, but the test behaves as expected if we change it to `???` (I can reproduce it correctly locally)

Reply:
Huh, maybe I'm misunderstanding, what I'm testing looks like this: https://github.com/aherlihy/scala3/compare/i15503...aherlihy:scala3:i16723-2?expand=1

Is that what you're running as well?

Reply:
Yes

Reply:
It does not pass for me, not sure why it would behave differently